### ISSUE TITLE
gcc-arm-none-eabi: non-standard scheme

### ISSUE THREAD

**Issue by blshkv** [repology/repology-rules#556]
https://repology.org/project/gcc-arm-none-eabi/versions

The latest version is detected as ```10-2020-q4``` currently, but this is not correct, because 10.3-2021.10 was released.

I reported a bug here:
https://bugs.launchpad.net/gcc-arm-embedded/+bug/1949004

For now, gcc-arm version is based on <upstream_version>-<arm_version>. Here is the list of last few versions FYI:
```
9-2020-q2-update
10-2020-q4-major
10.3-2021.07
10.3-2021.10
```
